Library/Homebrew/formula_creator.rb

```diff
@@ -5,49 +5,58 @@
 require "erb"
 
 module Homebrew
+  module_function
+
+  def url2name(url)
```

Comment:
This should be in the `FormulaCreator` class

Reply:
Then it should be marked as stateless, to be called without instantiating a new object. What is the right way to define static method in Ruby?

Reply:
`self.`

Reply:
Done.